sql/02-create-table-friend.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE friend (
+  id1 SERIAL references student (id),
```

Comment:
It's not best practice but this works in Postgres. Serial takes the place of primary key, and when creating the intersection table like this one, serial still works (but would break in MySQL)...so itd probably be better to use integer for this one.